### ISSUE TITLE
Phone number is not being shown displayed correctly

### ISSUE THREAD

**Issue by gnumilanix** [freshdesk/freshchat-android#5]
Freshchat website does not display the phone number correctly for all the numbers. One of the sample phone numbers _0987654321_ and is displayed correctly. but,  _0909999999_, which we tried to verify with  https://numverify.com/ and it's marked as valid. The country code for both of these numbers were _66 (Thailand)_

The way we are setting the phone number is with following:
``` java 
final FreshchatUser freshUser = Freshchat.getInstance(context.getApplicationContext()).getUser();
freshUser.setPhone("66", "0909999999");
Freshchat.getInstance(context.getApplicationContext()).setUser(freshUser);
```

**Displayed as expected**:
![screenshot from 2017-12-14 13-02-15](https://user-images.githubusercontent.com/1477917/34026652-77ad8b06-e192-11e7-9777-a9a02d4b52e7.png)

**Only displaying country code**:
![screenshot from 2017-12-15 12-17-33](https://user-images.githubusercontent.com/1477917/34026653-77dbca52-e192-11e7-9bc2-5aa0f921f4c1.png)

**Comment by tryprasannan** [comment]
@gnumilanix We checked this and found an issue with phone number on subsequent updates (first time update will work fine). And this has been fixed in [Version 1.2.2](https://github.com/freshdesk/freshchat-android/releases/tag/1.2.2), request you to upgrade.

**Comment by gnumilanix** [comment]
@tryprasannan verified and working on 1.2.2.
